internal/i18n/en_translations.go

```diff
@@ -189,4 +189,7 @@ var (
 	MsgRejected                    = ffm("FF10269", "Message with ID '%s' was rejected. Please check the FireFly logs for more information")
 	MsgConfirmQueryParam           = ffm("FF10270", "When true the HTTP request blocks until the message is confirmed")
 	MsgRequestMustBePrivate        = ffm("FF10271", "For request messages you must specify a group of private recipients", 400)
+	MsgUnknownTokensPlugin         = ffm("FF10272", "Unknown tokens plugin '%s'")
+	MsgMissingTokensPluginConfig   = ffm("FF10273", "Invalid tokens configuration - name and connector are required")
```

Comment:
Wonder if this is a little inconsistency in the use of the word `connector`.
Here we're using it to infer the transport type to talk between FireFly core, and the Tokens connector.
Probably `type`, or `transport` might be better 🤷  ... know there's on a limited number of names we can use for similar things here.

Reply:
We're really selecting which type of plugin to use (of which the only choice is currently "https", and which implies not only the transport but the structure of all the data going in and out of that API).